draft-tigress-requirements.md

```diff
@@ -121,45 +121,45 @@ General terms:
+- (Req-ArbitraryFormat) The solution SHALL support arbitrary message formats to support both keys that implement public standards like CCC as well as proprietary implementations of digital keys.
+- (Req-UnderstoodFormat) Both Sender application and Receiver application MUST be able to recognize the format.
+- (Req-Simplicity) Where possible, the system SHOULD rely on simple building blocks to facilitate adoption and implementation.
+- (Req-IntermediaryAttestation) If any Intermediary is required - it SHALL implement mechanisms to prevent abuse by share initiating device, verifying that the device is in good standing and content generated by the sender device can be trusted by the Intermediary. The trust mechanism could be proprietary or publicly verifiable ( e.g. WebAuthN).
+- (Req-RoundTrips) Solution SHALL allow for multiple round trips or multiple reads/writes between one set of Sender and Receiver devices.
+- (Req-ReceiverTrust) If any Intermediary is required - the Receiver device SHOULD evaluate the trustworthiness of the Intermediary using a list of trusted/approved intermediaries.
+- (Req-Preview) Solution SHOULD allow for extensibility and discoverable extensions (preview of share invitation).
+- (Req-RedemptionHandling) ShareURL SHOULD route Receiver to redeem Provisioning Information using the designated Credential Management Application (e.g. Wallet).
 
 # Review of existing solutions
 
 A number of existing solutions / protocols have been reviewed in order to be used for secure credential transfer based on the requirements: GSS-API, Kerberos, AWS S3, email, Signal. None of the existing protocols comply with the requirements; the effort of modifying the existing protocols has been accessed to be significantly higher than introducing a new solution to solve this problem.
 
 ## Arbitrary Messaging Channel (Email / WhatsApp / SMS / Signal / etc.)
 
-The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel, but that would not provide a good user experience. Users MAY need to copy and paste the Provisioning Information, or need a special application to handle some new file type. This violates [Req-SmoothUX].
-If multiple round trips were required the user would need to manually managing multiple payloads of Provisioning Information. This would be very hard for anyone non technical and greatly limit adoption. This violates [Req-NontechnicalUX].
+The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel, but that would not provide a good user experience. Users MAY need to copy and paste the Provisioning Information, or need a special application to handle some new file type. This violates (Req-SmoothUX).
+If multiple round trips were required the user would need to manually managing multiple payloads of Provisioning Information. This would be very hard for anyone non technical and greatly limit adoption. This violates (Req-Simplicity).
```

Comment:
I think this should stay as "Req-NontechnicalUX".

Reply:
reverted to Req-NontechnicalUX